src/Account/Account.php

```diff
@@ -5,25 +5,18 @@
 /**
  * @author Milan Matějček
  */
-class Fio
+class Account
 {
+	/** @var Bank */
+	private $account;
 
 	/** @var string */
 	private $token;
 
-	/** @var Bank */
-	private $account;
-
-	public function __construct($token, Bank $account)
+	public function __construct($account, $token = NULL)
```

Comment:
Object Bank validate account and give you prefix, account and banc code. Object old name Fio take Bank and token. Because Bank care for validation if you want send money to foreing account.

I wont merge it.